sdks/python/apache_beam/examples/complete/game/hourly_team_score.py

```diff
@@ -216,77 +137,70 @@ def expand(self, pcoll):
-                      default='gs://dataflow-samples/game/gaming_data*.csv',
+                      type=str,
+                      default='gs://apache-beam-samples/game/gaming_data*.csv',
                       help='Path to the data file(s) containing game data.')
-  parser.add_argument('--dataset',
-                      dest='dataset',
+  parser.add_argument('--output',
+                      type=str,
                       required=True,
-                      help='BigQuery Dataset to write tables to. '
-                           'Must already exist.')
-  parser.add_argument('--table_name',
-                      dest='table_name',
-                      default='hourly_team_score',
-                      help='The BigQuery table name. Should not already exist.')
+                      help='Path to the output file(s).')
   parser.add_argument('--window_duration',
                       type=int,
                       default=60,
                       help='Numeric value of fixed window duration, in minutes')
   parser.add_argument('--start_min',
-                      dest='start_min',
+                      type=str,
                       default='1970-01-01-00-00',
                       help='String representation of the first minute after '
                            'which to generate results in the format: '
                            'yyyy-MM-dd-HH-mm. Any input data timestamped '
                            'prior to that minute won\'t be included in the '
                            'sums.')
   parser.add_argument('--stop_min',
-                      dest='stop_min',
+                      type=str,
                       default='2100-01-01-00-00',
                       help='String representation of the first minute for '
                            'which to generate results in the format: '
                            'yyyy-MM-dd-HH-mm. Any input data timestamped '
                            'after to that minute won\'t be included in the '
                            'sums.')
 
-  known_args, pipeline_args = parser.parse_known_args(argv)
-
-  pipeline_options = PipelineOptions(pipeline_args)
-  pipeline_options.view_as(SetupOptions).save_main_session = True
-  with beam.Pipeline(options=pipeline_options) as p:
+  args, pipeline_args = parser.parse_known_args()
 
+  with beam.Pipeline(argv=pipeline_args) as p:
     (p  # pylint: disable=expression-not-assigned
-     | ReadFromText(known_args.input)
-     | HourlyTeamScore(
-         known_args.start_min, known_args.stop_min, known_args.window_duration)
-     | WriteWindowedToBigQuery(
-         known_args.table_name, known_args.dataset, configure_bigquery_write()))
+     | 'ReadInputText' >> beam.io.ReadFromText(args.input)
+     | 'HourlyTeamScore' >> HourlyTeamScore(
+         args.start_min, args.end_min, args.window_duration)
+     | 'FormatTeamScores' >> beam.ParDo(FormatTeamScores())
+     | 'WriteTeamScoreSums' >> beam.io.WriteToText(args.output)
```

Comment:
The behaviour here is different compared to Java example, which uses per window files.